homeassistant/components/tplink/button.py

```diff
@@ -45,6 +45,9 @@ class TPLinkButtonEntityDescription(
             breaks_in_ha_version="2025.4.0",
         ),
     ),
+    TPLinkButtonEntityDescription(
+        key="reboot",
```

Comment:
Device class restart, then you can also omit the translation key :)

Reply:
Great catch, I should've remembered that! Fixed now.

Reply:
This [is the test](https://github.com/home-assistant/core/blob/dev/tests/components/tplink/__init__.py#L170-L176) I was referring to in the comment on the same point in https://github.com/home-assistant/core/pull/127883#discussion_r1791827763. We could potentially drop this check but it's quite handy to have a test to ensure we haven't forgotten any translations. Perhaps it could be extended to see if a unique device class is present if the translation key is not there.